package.json

```diff
@@ -7,9 +7,8 @@
     "url": "git://github.com/CartoDB/cartodb.js.git"
   },
   "author": {
-    "name": "CartoDB",
-    "url": "http://cartodb.com/",
-    "email": "wadus@cartodb.com"
+    "name": "CARTO",
+    "url": "https://carto.com/docs/carto-engine/carto-js"
```

Comment:
Docs are changing soon and this URL will probably get outdated... We could perhaps just point to http://carto.com/?

Reply:
I didn't know that! Ok, will update.